firewood/src/merkle/node.rs

```diff
@@ -459,6 +463,203 @@ impl Storable for Node {
     }
 }
 
+pub struct EncodedNode<T: ?Sized> {
```

Comment:
Q: What cases are we using that aren't `Sized`?

Reply:
Good catch, I think this bound is not necessary.